sdk/eventhub/Microsoft.Azure.EventHubs/tests/Infrastructure/TestUtility.cs

```diff
@@ -13,16 +13,99 @@ namespace Microsoft.Azure.EventHubs.Tests
 
     internal static class TestUtility
     {
+        //Modified to fit Track two TestEnvironment.cs 
         private static readonly Lazy<string> EventHubsConnectionStringInstance =
-            new Lazy<string>( () => BuildEventHubsConnectionString(ReadEnvironmentVariable(TestConstants.EventHubsConnectionStringEnvironmentVariableName)), LazyThreadSafetyMode.PublicationOnly);
+            new Lazy<string>( () => ReadEnvironmentVariable(TestConstants.EventHubsConnectionStringEnvironmentVariableName), LazyThreadSafetyMode.PublicationOnly);
 
         private static readonly Lazy<string> StorageConnectionStringInstance =
             new Lazy<string>( () => ReadEnvironmentVariable(TestConstants.StorageConnectionStringEnvironmentVariableName), LazyThreadSafetyMode.PublicationOnly);
-
+        //Copied from Track two TestEnvironment.cs
```

Comment:
Please remove all references to copying code from elsewhere; it's not relevant for context of future maintainers.